src/types/runtime/gravity/OrderedSet.ts

```diff
@@ -27,6 +27,7 @@ export const OrderedSet = Record({
   display_on_martsy: Boolean,
   created_at: String,
   updated_at: String,
+  created_by: String,
```

Comment:
Is this right? (i'm a little fuzzy on this file). But `created_by` should be `Record({ name: ..., id: .... })` (all the embedded user JSON fields).

Seems onerous! I _think_ we can maybe skip this (or does it complain about resolving the `created_by` field)?

Reply:
If so, maybe you can remove it from here (since it's not really a `String` and it doesn't _need_ to be added to this file), and instead add an `created_by: any` to the declaration here: https://github.com/artsy/metaphysics/blob/691155fc14f2680f6e3bbd0d2d65bc5cffca5c07/src/schema/v2/OrderedSet/OrderedSet.ts#L24

Reply:
Nice catch! It was complaining about resolving the `created_by` field, so I added this when I was thinking it just returned back the `id` of the `user` and I was going to have to pass it in as the arg to the `userByIdLoader`, which is incorrect. Will fix it with your recommendation!